source/microbit/help.c

```diff
@@ -81,57 +81,64 @@ STATIC const mp_doc_t help_table_types[] = {
     {&microbit_accelerometer_type, "MicroBitAccelerometer type\n"},
 };
 
+// Consistency between messages and minimal jargon improves help text.
 STATIC const mp_doc_t help_table_instances[] = {
-    {&microbit_module, "A toolbox of code to control the micro:bit hardware.\n"},
-    {&microbit_panic_obj, "panic() -- enter panic mode (requires a restart).\n"},
-    {&microbit_random_obj, "random(n) -- return a random number between 0 and 'n'-1.\n"},
-    {&microbit_sleep_obj, "sleep(n) -- wait for 'n' milliseconds (1 second = 1000).\n"},
-    {&microbit_running_time_obj, "running_time() -- get the number of milliseconds since reset.\n"},
-    {&microbit_accelerometer_obj, "A toolbox of code to detect the device's position (orientation).\n"},
-    {&microbit_accelerometer_get_x_obj, "get_x() -- get the device's X axis reading.\n"},
-    {&microbit_accelerometer_get_y_obj, "get_y() -- get the device's Y axis reading.\n"},
-    {&microbit_accelerometer_get_z_obj, "get_z() -- get the device's Z axis reading.\n"},
-    {&microbit_button_a_obj, "Represents button A. Use its is_pressed() method to discover its status.\n"},
-    {&microbit_button_b_obj, "Represents button B. Use its is_pressed() method to discover its status.\n"},
-    {&microbit_button_is_pressed_obj, "is_pressed() -- True if the button is pressed, else False.\n"},
-    {&microbit_compass_obj, "A toolbox of code to detect the device's heading.\n"},
+    {&microbit_module, "A group of tools to control the micro:bit hardware.\n"},
+    // System state objects
+    {&microbit_panic_obj, "Begin panic() mode (requires a reset).\n"},
```

Comment:
I like how the usage is now part of the sentence rather than at the start. :-)

Trying to work out how we can explain why one would need to go into panic mode. How about... "Go panic() when something is wrong (requires a reset)."